tutorials/editor/default_key_mapping.rst

```diff
@@ -359,7 +359,7 @@ Script editor
 +----------------------+---------------------------------+---------------------------------+----------------------------------------+
 | Save All             | :kbd:`Ctrl + Shift + Alt + S`   | :kbd:`Cmd + Shift + Opt + S`    | ``script_editor/save_all``             |
 +----------------------+---------------------------------+---------------------------------+----------------------------------------+
-| Soft Reload Script   | :kbd:`Ctrl + Shift + R`         | :kbd:`Cmd + Shift + R`          | ``script_editor/reload_script_soft``   |
+| Soft Reload Script   | :kbd:`Ctrl + Alt + R`           | :kbd:`Opt + Cmd + R`            | ``script_editor/reload_script_soft``   |
```

Comment:
```suggestion
| Soft Reload Tool Script   | :kbd:`Ctrl + Alt + R`           | :kbd:`Opt + Cmd + R`            | ``script_editor/reload_script_soft``   |
```
So unfortunately, the name of the shortcut has changed and would need to be updated. That also means every other box on this section would need to be updated as well. I don't want to merge this without the rename, but this might be better left for a PR that goes through everything to clean it up. I'm sure more of these are out of date, and also not in the same order in the editor anymore.